deploy/helm/edot-collector/kube-stack/values.yaml

```diff
@@ -574,28 +549,11 @@ collectors:
           #   insecure_skip_verify: true
           mapping:
             mode: otel
-        elasticsearch/ecs:
-          endpoints:
-            - ${env:ELASTIC_ENDPOINT}
-          api_key: ${env:ELASTIC_API_KEY}
-          mapping:
-            mode: ecs
       service:
         pipelines:
           metrics:
             receivers: [otlp]
-            exporters: [routing]
-          metrics/infra/ecs:
-            receivers: [routing]
-            processors:
-              - elasticinframetrics
-              - attributes/dataset
-              - resource/process
-              - batch/metrics
-            exporters: [debug, elasticsearch/ecs]
-          metrics/otel:
-            receivers: [routing]
-            processors: [batch]
```

Comment:
The metrics pipeline used to use the regular `batch` processor configuration, which is now changed to `batch/metrics`. Can you confirm that this is intentional?

Reply:
Correct, metrics batcher should never split metrics requests, thus I took the opportunity to change it here. Context: https://github.com/elastic/elastic-agent/pull/6928